### PR TITLE
ELEV-5957-sign-up-ip

### PR DESCRIPTION
**Pull request by peter987bp**

### CODE REVIEW

File: src/components/SignUpForm.js [hunk 1]
```diff
@@ -12,377 +12,391 @@ import Datetime from "elevate-ui/Datetime";
...
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            formState: null,
+        };
+    }
+
+    componentDidMount() {
+        const url = 'http://localhost:8005/api/marketing-site/ip';
```
Comment (vmera21) [inline src/components/SignUpForm.js:23]: Here we need to have some sort of an environment based url

File: .idea/vcs.xml
Comment (jdomonell) [inline .idea/vcs.xml:1]: i think you can git ignore this, since it's a local .idea config

File: src/components/SignUpForm.js [hunk 2]
```diff
@@ -12,377 +12,391 @@ import Datetime from "elevate-ui/Datetime";
...
+        fetch(url, {
+            method: 'POST',
+            credetials:'include',
```
Comment (jdomonell) [inline src/components/SignUpForm.js:26]: please fix to "credentials"

File: src/components/SignUpForm.js (hunk 1 above)
Comment (jdomonell) [reply src/components/SignUpForm.js:23]: this const should be a little more descriptive

**Review by jdomonell — CHANGES_REQUESTED** [review]
a few comments so far

**Review by jdomonell — APPROVED** [review]

**Pull request merged**